src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SecurityServerTlsCertificate/SecurityServerTlsCertificate.vue

```diff
@@ -169,6 +183,7 @@ export default Vue.extend({
       this.$router.push({
         name: RouteName.GenerateInternalCSR,
       });
+      // this.showGenCsr = true;
```

Comment:
In general commented code should be removed but this line seems to be added in this file and I can't see that statement with a red background so why is this added in this PR? If it's not needed it should be removed.

Reply:
Removed the line